lib/fizzy/execute.cpp

```diff
@@ -687,7 +688,8 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             const auto br_table_size = read<uint32_t>(immediates);
             const auto arity = read<uint32_t>(immediates);
 
-            const auto br_table_idx = stack.pop();
+            // TODO: is uint64_t the correct stack type here?
+            const auto br_table_idx = stack.pop().as<uint64_t>();
```

Comment:
```suggestion
            const auto br_table_idx = stack.pop().i64;
```
Here and in other places.